types/v56/nsxt_types.go

```diff
@@ -704,6 +704,7 @@ type NsxtAlbController struct {
 	// LicenseType By enabling this feature, the provider acknowledges that they have independently licensed the
 	// enterprise version of the NSX AVI LB.
 	// Possible options: 'BASIC', 'ENTERPRISE'
+	// This field was removed since VCD 10.4.0 (v37.0) in favor of NsxtAlbServiceEngineGroup.SupportedFeatureSet
 	LicenseType string `json:"licenseType,omitempty"`
```

Comment:
I had doubts about changing to pointer here, but as the newest VCD doesn't complain about sending/receiving it, I didn't.
That's the main reason `SupportedFeatureSet` is a pointer, avoids breaking <v37.0

Reply:
I think we can avoid adding a pointer (which could cause someone an unnecessary change when upgrading SDK ).

Go JSON lib does not send `string` type with empty value `""` when `omitempty` is present.